lumibot/tools/indicators.py

```diff
@@ -9,7 +9,7 @@
 import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
-import quantstats as qs
+import quantstats_lumi as qs
```

Comment:
Did you mean for this to be added as a subdir? If so, it was never added to your git commit